trustfall_core/src/interpreter/hints/vertex_info.rs

```diff
@@ -14,6 +14,17 @@ use crate::{
 
 use super::{dynamic::DynamicallyResolvedValue, CandidateValue, EdgeInfo, Range};
 
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct RequiredProperty {
```

Comment:
I need to re-home this, it's just on the same file as the `VertexInfo` for convenience when I was coding it

Reply:
Hum... I'm not sure where else this would better fit, what do you think @obi1kenobi ?

Reply:
I think it's fine here. Just make sure to re-export it wherever `VertexInfo` itself is re-exported, including in the `trustfall` crate?

Reply:
Ah, I totally missed that! Exported it on those places